image_provisioner/playbooks/roles/docker-save/files/docker_save.sh

```diff
@@ -0,0 +1,7 @@
+#/bin/bash
```

Comment:
there is an atomic save command...can you please look at that to see if it can replace this script?